public/js/controllers/ticketpool_controller.js

```diff
@@ -75,14 +74,17 @@ function priceGraphData (items, memP) {
   return p
 }
 
-function getVal (val) { return isNaN(val) ? 0 : val }
-
-function outputsGraphData (items) {
-  return [
-    getVal(items.solo),
-    getVal(items.pooled),
-    getVal(items.txsplit)
-  ]
+function populateOutputs (data) {
+  var totalCount = 0
+  data.count.map((n) => { totalCount += n })
+  var tableData = '<thead><td># of Outputs</td><td>Count</td><td>% Occurence</td></thead>'
+  data.outputs.map((n, i) => {
+    var count = data.count[i]
+    tableData += '<tr><td class="pr-2 lh1rem vam nowrap xs-w117 font-weight-bold">' + n + '</td>' +
+    '<td><span class="hash lh1rem"> ' + count + ' </span></td>' +
+    '<td><span class="hash lh1rem"> ' + ((count * 100) / totalCount).toFixed(4) + '% </span></td></tr>'
+  })
```

Comment:
We are trying to use template literals for this kind of thing.

Also, anytime we are mixing HTML with network data we should sanitize the HTML before inserting. In this case, since a floating `thead` is not valid HTML, you'll want to create a node and sanitize `{ IN_PLACE: true }`, otherwise `dompurify` will sometimes behave weirdly. 

Alternatively, you could just pass your values through a `parseInt` or `parseFloat`, since nothing should be a string.